constantine/ethereum_verkle_ipa.nim

```diff
@@ -167,8 +196,20 @@ func serialize*(dst: var EthVerkleIpaProofBytes,
   a0[].serialize_fr(src.a0, littleEndian)
   return cttEthVerkleIpa_Success
 
-func deserialize*(dst: var EthVerkleIpaProof,
-                  src: EthVerkleIpaProofBytes): cttEthVerkleIpaStatus =
+func serialize_aff*(dst: var EthVerkleIpaProofBytes,
+                src: EthVerkleIpaProof
+                ): cttEthVerkleIpaStatus {.libPrefix: prefix_ffi.} =
+  
+  return serialize(dst, src.aff)
+
+func serialize_prj*(dst: var EthVerkleIpaProofBytes,
+                src: EthVerkleIpaProof
+                ): cttEthVerkleIpaStatus {.libPrefix: prefix_ffi.} =
+  
+  return serialize(dst, src.prj)
```

Comment:
A user will not know what serialization to use between prj and aff, this will lead to errors and so is not the correct API.